python/grass/script/tests/grass_script_setup_test.py

```diff
@@ -79,3 +79,44 @@ def init_finish(queue):
     session_file = run_in_subprocess(init_finish)
     assert session_file, "Expected file name from the subprocess"
     assert not os.path.exists(session_file), "Session file not deleted"
+
+
+@pytest.mark.usefixtures("mock_no_session")
+def test_init_as_context_manager_env_attribute(tmp_path):
```

Comment:
Is there a reason the fixture isn't directly used (in pytest it is an argument, like `tmp_path`), but uses `@pytest.mark.usefixtures("mock_no_session")`?

Reply:
Yes, mock_no_session is unused, so it would trigger an unused variable warning. I can document that, but then, will we document that every time this is needed? Maybe? I don't want to disable code quality warnings for tests just because they are tests.

Reply:
Ok, so maybe just a note in the fixture to say that it must be used that way to avoid _X_ warning would be fine to keep the knowledge

Reply:
Done.